coalib/bearlib/abstractions/Linter.py

```diff
@@ -96,6 +96,9 @@ def _prepare_options(options):
         raise ValueError("Invalid `output_format` specified.")
 
     if options["prerequisite_check_command"]:
+        logging.warn("The bear which uses the following executable --{} "
+                     "uses ``prerequisite_check_command`` which is "
+                     "depreciated.".format(options["executable"]))
         if "prerequisite_check_fail_message" in options:
             assert_right_type(options["prerequisite_check_fail_message"],
                               str,
```

Comment:
btw, when you deprecate the option, you should have an alternative :)
You can do another PR imo implementing the stuff mentioned in the issue, but please remove the "Fixes xxxx" reference, as you don't close the issue completely :)